tests/page/page-aria-snapshot.spec.ts

```diff
@@ -421,3 +421,41 @@ it('should treat input value as text in templates', async ({ page }) => {
     - textbox: hello world
   `);
 });
+
+it('should respect aria-owns', async ({ page }) => {
+  await page.setContent(`
+    <a href='about:blank' aria-owns='input p'>
+      <div role='region'>Link 1</div>
+    </a>
+    <a href='about:blank' aria-owns='input p'>
+      <div role='region'>Link 2</div>
+    </a>
+    <input id='input' value='Value'>
+    <p id='p'>Paragraph</p>
+  `);
+
+  // - Different from Chrome DevTools which attributes ownership to the last element.
```

Comment:
Why not align on this item with devtools?

Reply:
That's unfortunately not easy at all. We could probably throw in this case, but why?